Environment_Engine/Query/FloorGeometry.cs

```diff
@@ -42,8 +42,9 @@ public static partial class Query
 
         [Description("Returns the floor geometry of a space represented by Environment Panels as a BHoM Geometry Polyline")]
         [Input("panelsAsSpace", "A collection of Environment Panels that represent a closed space")]
+        [Input("tolerance", "Distance tolerance for planar checks, default set to BH.oM.Geometry.Tolerance.Distance")]
```

Comment:
The tolerance in this instance is actually for the `BooleanUnion` of the floor lines, rather than planar checks :smile: